README.md

```diff
@@ -137,8 +137,13 @@ If you're here to implement a lexer for your awesome language, there's a good ch
 
 ### Run the tests
 
-You can test the core of Rouge simply by running `rake` (no `bundle exec` required), or `rake spec TEST=spec/xxx_spec.rb`
-to run a single test file.
+You can test the core of Rouge simply by running `rake` (no `bundle exec` required). You can also run a single test file by
```

Comment:
Because `tasks/changelog.rake` includes a `require` call as the first line, `bundle exec` is actually required (at the moment at least) as merely loading the Rake files throws an error.

What do you think is the best way to handle this? Change this line? The `profile_memory` task avoids this problem by having its call to `require` occur within the task block but it still fails with a somewhat cryptic error if you try to run `rake profile_memory`.

My inclination is to change this line.

Reply:
> it still fails with a somewhat cryptic error if you try to run `rake profile_memory`

Can you post the full error message (with stack trace, if available). I can run `rake profile_memory` on Ruby 2.4 without any issues.

Reply:
Sorry, to be clear, it's not a cryptic error message to me. I know it's because I don't have the gem installed.

```
$ rake profile_memory
rake aborted!
LoadError: cannot load such file -- memory_profiler
rouge/Rakefile:47:in `block in <top (required)>'
Tasks: TOP => profile_memory
(See full trace by running task with --trace)
```

If you're not seeing it on your system, it's because you've got memory_profiler installed.

Reply:
This was addressed by #1242.